package.json

```diff
@@ -45,7 +46,7 @@
         "safe-url-assembler": "^1",
         "sharp": "^0",
         "tailwindcss": "^3",
-        "typescript": "^5",
+        "typescript": "^5.5.0",
```

Comment:
Why this change? Any reason?

Reply:
I changed it for testing purposes, bc lint somehow warned about not supporting 5.6.0 and above... but honestly I don't care, as it's working